### DISCUSSION TITLE
Fooocus help!

### DISCUSSION THREAD

**Discussion by GodMollo7** [community/community#80975] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm having some issues when trying to launch Fooocus. I followed all of the steps but is still doesn't open. I'm using my Macbook Pro

**Comment by ghostinhershell** [comment]
Welcome to the GitHub Community, @GodMollo7, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
